### DISCUSSION TITLE
How to use Babel Macros in production builds

### DISCUSSION THREAD

**Discussion by maor-benami** [emotion-js/emotion#2893] [Q&A]
I am using NextJS.
I am also using this `.babelrc` config:
`
{
  "presets": ["next/babel"],
  "plugins": ["babel-plugin-macros", "@emotion/babel-plugin", "inline-react-svg"]
}
`

I would like to keep class-names in production and could not find how to do it in the docs.
In production I get hashed class names

Thank you

**Comment by srmagura** [comment]
I assume you mean you want to keep automatic labels in your class names in production. You can do this by setting the [`autoLabel` option](https://emotion.sh/docs/@emotion/babel-plugin#autolabel) to `"always"` in the Babel plugin's configuration.

Also, I don't think `babel-plugin-macros` is necessary when using `@emotion/babel-plugin`.
